client/src/Components/Icon/Icon.js

```diff
@@ -1,9 +1,10 @@
 import pfp from "./pfpicon.png";
 import stylesheet from './Icon'
 
-const IconNavBar = () => {
+const IconNavBar = (props) => {
+  const { setSettingsFocus } = props;
     return(
-      <img className="icon" src={pfp} style={{stylesheet}}></img>
+      <img className="icon" src={pfp} style={{stylesheet}} onClick={() => setSettingsFocus(true)}></img>
```

Comment:
if possible change this line to 

`<img className="icon" src={pfp} style={{stylesheet}} onClick={() => setSettingsFocus(true)}/>`

Reply:
Done.